### ISSUE TITLE
No rule for space after stab

### ISSUE THREAD

**Issue by saverio-kantox** [bbatsov/ruby-style-guide#603]
There is no rule for the separation between the stab and its parameters. The parser accepts both with space and without space but it seems that in any other language the 'arrow-like' preferred syntaxes are with space.
```ruby
# Bad
->(x) { x }
# Good
-> (x) { x }
```

references: 
http://eslint.org/docs/rules/arrow-spacing
https://google.github.io/styleguide/javaguide.html

**Comment by joshminnie** [comment]
I agree with adding this rule, at least from a consistency standpoint. In many of the projects I've worked in, I've seen both flavors. I don't have a strong preference as to which one should be the default behavior.

**Comment by RomanADavis** [comment]
Without a space makes more sense to me, the same way you would use parenthesis around the argument of any other method or function, because that's what a lambda _is_, just without a name.

**Comment by bbatsov** [comment]
I'd mention the version with space didn't work in Ruby 1.9 when this syntax was introduced, so a lot of code was written without a space and probably we should stick to this for consistency.

**Comment by TheSmartnik** [comment]
@bbatsov we do have an example of [incompatible with 1.9 rule](https://github.com/bbatsov/ruby-style-guide#percent-i). Why not make the same note here as well?

**Comment by bbatsov** [comment]
My point is not 1.9 compatibility, but the fact that originally the suggested good syntax didn't work and therefore a lot of code was written without a space. I myself am more used to this syntax for that very reason. I guess someone should check which syntax is more popular and we can just suggest sticking to it.
